misc/snapcraft_releases.py

```diff
@@ -1,6 +1,6 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 """
-For a given Parsec version, this script outputs the `--release=<channel>` options intended for snapcraft.
+For a given Parsec version, this script outputs the `<channel>` value intended to be used in the `--releases=<channel>` option for `snapcraft upload`.
```

Comment:
Approved too quickly ^^
```suggestion
For a given Parsec version, this script outputs the `<channel>` value intended to be used in the `--release=<channel>` option for `snapcraft upload`.
```